ikarus/io/resultfunction.hh

```diff
@@ -82,7 +82,7 @@ public:
     if constexpr (std::is_same_v<UserFunction, Impl::DefaultUserFunction>) {
       Dune::FieldVector<ctype, griddim> val(0.0);
 
-      auto sigma = fes_->at(0).template calculateAt<resType>(feRequirements_, val);
+      auto sigma = fes_->at(0).template calculateAt<resType>(feRequirements_, val).asVec();
```

Comment:
If `asVec` is hard coded here, then the `return static_cast<int>(sigma.rows() * sigma.cols());` can be simplified

Reply:
Yeah I would for now leave it hardcoded

Reply:
Ah no wait this function is now not even needed anymore. You can calculate the vector size at compile time

Reply:
But not if we have dymaic size result types right?

Reply:
Ah yes you are right.